twitter_delete.rb

```diff
@@ -96,22 +98,33 @@ def api_call(method, *args)
   tweets_to_delete += tweets.reject(&method(:too_new_or_popular?))
 end
 
+puts "==> Checking timeline to API Completed"
+
 if @options[:archive_given]
-  puts "==> Checking archive JS..."
+  puts "==> Checking archive JS part = ".concat(@options[:part].to_s)
+  
   archive_tweet_ids = []
 
   # tweet.js is not valid JSON...
+  puts "==> Fixing tweet.js is not valid JSON"
   file_contents = File.read(@options[:archive])
-  file_contents.sub! "window.YTD.tweet.part0 = ", ""
+  file_contents.sub! "window.YTD.tweet.part".concat(@options[:part].to_s).concat(" = "), ""
```

Comment:
Can you explain this a bit more?

Reply:
Refer from my tweets archive files, it's large lists of tweets get split up into multiple files, which are all found in the same place.  

So as well as "tweet.js" (window.YTD.tweet.part0) I might have "tweet-part1.js" (window.YTD.tweet.part1) , "tweet-part2.js" (window.YTD.tweet.part2) etc.

Reply:
Ok, makes sense. I think ideally this wouldn't require input from the user but instead would read and iterate through all these values. How about:

```suggestion
  file_contents.gsub! /window\.YTD\.tweet\.part\d+ = /, ""
```